Yumemi-iOS-Training/Model/WeatherModel.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  WeatherModel.swift
+//  Yumemi-iOS-Training
+//
+//  Created by 成尾 嘉貴 on 2022/06/21.
+//
+
+import Foundation
+import YumemiWeather
+
+struct WeatherModel {
+    private var dateFormatter: DateFormatter {
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ssZZZZZ"
+        return dateFormatter
+    }
+    
+    func jsonString(request: WeatherRequest) throws -> String{
+        let encoder = JSONEncoder()
+        encoder.dateEncodingStrategy = .formatted(dateFormatter)
+        let requestJsonValue = try encoder.encode(request)
+        guard let requestJsonString = String(bytes: requestJsonValue, encoding: .utf8) else{
```

Comment:
```suggestion
        guard let requestJsonString = String(bytes: requestJsonValue, encoding: .utf8) else {
```
String(bytes:) はC言語を取り組む時使うもの、普通は [init(data:encoding:)](https://developer.apple.com/documentation/swift/string/init(data:encoding:)) 使います。